terraform/shard-core/create-shard-core-oracle.sh

```diff
@@ -22,7 +22,7 @@ LOCAL_PATH=$(dirname "${BASH_SOURCE[0]}")
 
 [ -z "$RELEASE_NUMBER" ] && RELEASE_NUMBER=0
 
-[ -z "$VISITORS_ENABLED" ] && VISITORS_ENABLED="false"
+[ -z "$VISITORS_FACTOR" ] && VISITORS_FACTOR=1
```

Comment:
```suggestion
[ -z "$DEFAULT_VISITORS_FACTOR" ] && DEFAULT_VISITORS_FACTOR=1
[ -z "$VISITORS_FACTOR" ] && VISITORS_FACTOR=$DEFAULT_VISITORS_FACTOR
```
This will allow us to set DEFAULT_VISITORS_FACTOR in the stack-env for an environment